server/graphcool.yml

```diff
@@ -34,8 +34,11 @@ functions:
 # Read more here:
 # https://docs-next.graph.cool/reference/auth/authorization/overview-iegoo0heez
 permissions:
-- operation: "*"
-
+# - operation: "*"
+  - operation: User.read
+  - operation: User.update
+    authenticated: true
+    query: ./src/auth0/updateUser.graphql
```

Comment:
At this point, what does this query get us over just `authenticated: true`?

Reply:
This query makes sure that the User sending the mutation (`$user_id`) is the User being modified (`$input_id`).